templates/portfolio/Tutorial/styles.css

```diff
@@ -0,0 +1,105 @@
+
+.portfolio-header {
+  position: relative;
+  background-color: #D8D8D8;
+  background-image: url(images/header-bg.jpg);
+}
+
+.mdl-layout__header-row {
+  padding: 0;
+  justify-content: center;
+}
+
+.portfolio-navigation-row {
+  background-color: rgba(0, 0, 0, 0.08);
+  text-transform: uppercase;
+  height: 45px;
+}
+
+.portfolio-navigation-row  .mdl-navigation {
+  text-align: center;
+  max-width: 900px;
+  width: 100%;
+}
+
+.portfolio-navigation-row .mdl-navigation__link {
+  flex: 1;
+  line-height: 42px;
+}
+
+.mdl-layout__header .mdl-layout__drawer-button {
+    background-color: rgba(197, 197, 197, 0.44);
+}
+
+.portfolio-navigation-row .is-active {
+  position: relative;
+  font-weight: bold;
+}
+
+.portfolio-navigation-row .is-active:after {
+  content: "";
+  width: 70%;
+  height: 2px;
+  display: block;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  background-color: rgb(255,64,129);
+  left: 15%;
+}
+
+
+img {
```

Comment:
Consider adding a class to your images and using it here. Targeting an element like this is also pretty dangerous.